### PR TITLE
Set default project path for localfs in project form

### PR DESCRIPTION
**Pull request by phlax**
as in title

ref #6639

### CODE REVIEW

File: pootle/apps/pootle_app/forms.py [hunk 1]
```diff
@@ -93,8 +95,13 @@ def clean_code(self):
         return self.cleaned_data['code'].strip()
 
     def save(self, commit=True):
+        created = bool(not self.instance.pk)
         project = super(ProjectForm, self).save(commit=commit)
         project.config["pootle_fs.fs_type"] = self.cleaned_data["fs_plugin"]
+        if created and self.cleaned_data["fs_plugin"] == "localfs":
```
Comment (unho) [inline pootle/apps/pootle_app/forms.py]: What if an already existing project changed its plugin to `localfs`. I would just drop the `created` bit.

Comment (phlax) [reply pootle/apps/pootle_app/forms.py]: if we do that it will overwrite any custom path

Comment (unho) [reply pootle/apps/pootle_app/forms.py]: Sure, but that is what we want, isn't it?

Comment (phlax) [reply pootle/apps/pootle_app/forms.py]: i mean it would overwrite an already configured project

Comment (unho) [reply pootle/apps/pootle_app/forms.py]: mmm, you are right. But the first part of my initial comment in this thread still stands.

Comment (phlax) [reply pootle/apps/pootle_app/forms.py]: if you change the fs_type then it would be kinda up to you to update the url too. This is just setting a default

**Comment by phlax** [comment]
obsoleted by #6710

**Pull request closed without merging**